src/components/explorer/ExplorerIndividualContent.js

```diff
@@ -62,6 +67,33 @@ const ExplorerIndividualContent = () => {
   const resourcesTuple = useIndividualResources(individual);
   const individualContext = useMemo(() => ({ individualID, resourcesTuple }), [individualID, resourcesTuple]);
 
+  const dispatch = useAppDispatch();
+
+  const biosamplesData = useDeduplicatedIndividualBiosamples(individual);
+
+  const allExperimentResults = useMemo(
+    () => biosamplesData.flatMap((b) => (b?.experiments ?? []).flatMap((e) => e?.experiment_results ?? [])),
+    [biosamplesData],
+  );
+
+  const viewableExperimentResultsForIgv = useIndividualViewableExperimentResults(individual);
```

Comment:
move this line to above the effect that uses it since they're linked conceptually